src/Cli.Tests/TestHelper.cs

```diff
@@ -1133,5 +1133,25 @@ public static void SetupTestLoggerForCLI()
             ConfigGenerator.SetLoggerForCliConfigGenerator(configGeneratorLogger.Object);
             Utils.SetCliUtilsLogger(utilsLogger.Object);
         }
+
+        /// <summary>
+        /// Creates basic initialization options for MS SQL config.
+        /// </summary>
+        /// <param name="config">Optional config file name.</param>
+        /// <returns>InitOptions</returns>
+        public static InitOptions CreateBasicInitOptionsForMsSqlWithConfig(string? config = null)
+        {
+            return new(
+                databaseType: DatabaseType.mssql,
+                connectionString: "testconnectionstring",
+                cosmosNoSqlDatabase: null,
+                cosmosNoSqlContainer: null,
+                graphQLSchemaPath: null,
+                setSessionContext: true,
+                hostMode: HostModeType.Development,
+                corsOrigin: new List<string>(),
+                authenticationProvider: EasyAuthType.StaticWebApps.ToString(),
```

Comment:
Do other sample configs also set the authentication provider to `StaticWebApps`?

Reply:
Yep, all the test configs use `StaticWebApps`